tests/unit/test_platforms_get_platform.py

```diff
@@ -196,23 +196,6 @@ def test_get_platform_using_platform_from_job_info(
     assert get_platform(task_conf)['name'] == expected_platform_name
 
 
```

Comment:
@MetRonnie with this test I think the Codecov warning above would go away.

```py
def test_get_platform_from_config_with_subshell(mock_glbl_cfg):
    # A platform definition is using subshell:
    mock_glbl_cfg(
        'cylc.flow.platforms.glbl_cfg',
        '''
        [platforms]
            [[mace]]
                hosts = mace001, mace002
                job runner = slurm
        '''
    )
    task_conf = {'platform': '$(host)'}
    platform = get_platform(task_conf)
    assert platform is None
```

(Not sure if `mock_glbl_cfg` is necessary or needs changing too, but just to test that option that returns None) :+1:

Reply:
Added here https://github.com/cylc/cylc-flow/pull/4152/commits/969f758dc176efc4e4d362a0b690c9ea62839dfa